src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3984,11 +3984,18 @@ bool ApplicationManagerImpl::ResetVrHelpTitleItems(
 
   const std::string& vr_help_title = get_settings().vr_help_title();
   smart_objects::SmartObject so_vr_help_title(vr_help_title);
-
   app->reset_vr_help_title();
-  app->reset_vr_help();
   app->set_vr_help_title(so_vr_help_title);
 
+  app->reset_vr_help();
+  smart_objects::SmartObjectSPtr so_vr_help =
+      MessageHelper::CreateAppVrHelp(app);
+  if (!so_vr_help->keyExists(strings::vr_help)) {
+    SDL_LOG_WARN("Failed to create vr_help items");
+    return false;
```

Comment:
Eh, I don't know whether we should return false here, since we did at least clear out `vr_help`. I will note that this case doesn't seem possible at the moment, so I suppose it doesn't matter too much